nevergrad/functions/unit_commitment/core.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import nevergrad as ng
+from ..base import ExperimentFunction
+
+class UnitCommitmentProblem1(ExperimentFunction):
```

Comment:
is there a better name than `Problem1` ?
I feel that either the name should be more explicit, or that the `1` should be a parameter of the init so that you can instantiate all problems similarly.